sktime/forecasting/structural.py

```diff
@@ -0,0 +1,485 @@
+# -*- coding: utf-8 -*-
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Wraps the UnobservedComponents (state space) model from statsmodels."""
+
+__all__ = ["UnobservedComponents"]
+__author__ = ["Juan Orduz"]
```

Comment:
We try to use GitHub handles, that way we can eventually link them in our [estimator overview table](https://www.sktime.org/en/latest/estimator_overview.html)
```suggestion
__author__ = ["juanitorduz"]
```